project1.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="project1.css" type="text/css">
+  <link href="https://font.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <title>Setup and Mobile Version Skeleton</title>
+</head>
+
+<body>
+  <header>
+<!--<section>
+  <a><img src="Snapshoot Portfolio.png" alt=" "></a>
+<h1>Multi-Post Stories</h1>
+<ul>
+<li> CANOPY </li>
+<li>Back End Dev</li>
+<li>2015</li>
+</ul>
+<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<button type="button">html</button>
+<button type="button">css</button>
+<button type="button">javaScript</button>
+
+
+<button type="button">See Project</button>
+    
+</section>
+-->
+
+<section>
+
+<div class="flex-container1">
+
+<div class="snapshoot">
+  <a><img src="Snapshoot Portfolio.png" alt=" "></a>
+</div>
+
+<div class="flex-container12">
+<div class="one1"><h1>Multi-Post Stories</h1>
+</div>
+
+<div class="two2">
+
+<div> CANOPY </div>
+<div> </div>
+<div> Back End Dev </div>
+<div> </div>
+<div> 2015 </div>
+
+</div>
+
+<div class="three3"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+
+<div class="four4">
+<div><button>html</button></div>
+<div></div>
+<div></div>
+<div><button>css</button></div>
+<div></div>
+<div></div>
+<div><button>JavaScript</button></div>
+</div>
```

Comment:
- [ ] [REQUIRED] Please make changes! The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`.